rules/align.smk

```diff
@@ -45,16 +45,16 @@ def input_fq_args(fastqs):
 rule hisat2_align_bam:
     input:
         "data/ensembl/202122.1.ht2",
-        fq1="data/{sra}_1.fastq",
-        fq2="data/{sra}_2.fastq",
+        fq1="data/{sra}_1.fastq" if 'sra' in config and config["sra"] is not None else expand("data/{fq1}_1.fastq", fq1=config["fq1"]),
```

Comment:
Such a simple little solution. Thanks!

Reply:
Maybe add a case for `or len(config["sra"]) <= 0` for if it's just an empty list.